### ISSUE TITLE
ContainerBox container desktop gui - no fullscreen - any attempt to drag to larger size terminates that session

### ISSUE THREAD

**Issue by bmullan** [AlexandreDey/ContainerBox#10]
When I execute:

ContainerBox start -n <container_name> -f no

The non-fullscreen Desktop pops up as expected.

However, I noticed if I attempt to resize that Desktop window in any way... it (ie the desktop window) terminates

**Comment by AlexandreDey** [comment]
Yes it's a problem I have also encountered. Looking at the logs, it seems to be a Xephyr problem. I will try to see if changing some parameters solve the problem
